src/main/java/org/mongojack/MapReduce.java

```diff
@@ -250,11 +250,6 @@ private MapReduceCommand(String map, String reduce,
                 command.setScope(scope);
             }
             command.setVerbose(verbose);
-            if (extra != null) {
```

Comment:
We should probably also remove the variable `extra` then or else there's an option around that doesn't do anything...